backend/visualization/wordcloud.py

```diff
@@ -1,19 +1,17 @@
 from collections import Counter
 from sklearn.feature_extraction.text import CountVectorizer
 
-from addcorpus.python_corpora.load_corpus import load_corpus_definition
-from addcorpus.es_settings import get_stopwords_from_settings
+from addcorpus.models import Corpus
+from addcorpus.es_settings import get_nltk_stopwords
 from es import download as download
 
-def field_stopwords(corpus_name, field):
-    corpus = load_corpus_definition(corpus_name)
-    field_definition = next((f for f in corpus.fields if f.name == field), None)
-    mapping = field_definition.es_mapping
-    analyzer = mapping.get(
-        'fields', {}).get('clean', {}).get('analyzer')
-    if not analyzer:
+def field_stopwords(corpus_name, field_name):
+    corpus = Corpus.objects.get(name=corpus_name)
+    field = corpus.configuration.fields.get(name=field_name)
+    if field.language and field.language is not 'dynamic':
+        return get_nltk_stopwords(field.language)
```

Comment:
Ah, I see now why you'd want the `field.language` to be set. For mixed corpora, this would fall back to the first language of the `corpus.languages` array? So we'd still remove Finnish stopwords from Swedish texts. I do think we should also have an option to skip stopword removal altogether, but this can be left for later.

Reply:
In the wordcloud here, there is no fallback, so if the field has no language set (or it's "dynamic", meaning it's mixed but the value is included in the document), you don't remove stopwords.

I think that's more appropriate as the default behaviour, but I can imagine that for some corpora, it would be acceptable to remove stopwords based on the majority language.

Reply:
If you're really ambitious, the `'dynamic'` setting could also be used to remove finnish stopwords in finnish documents, and swedish stopwords in swedish documents.

That's a lot more complicated though, so it's not really in the scope of this PR.

Reply:
Hm, I'm getting a bit confused by `field.language` (see my [question](https://github.com/UUDigitalHumanitieslab/I-analyzer/issues/1522)), because the code here seems to indicate it can be an empty string, while the validator associated with the field seems to throw a ValidationError for values which are not "dynamic", or a valid language code.

Reply:
I totally agree with *not* getting overcomplicated here with stopword removal, and just leave that step our for mixed corpora.